src/EditorFeatures/VisualBasicTest/Recommendations/Statements/EachKeywordRecommenderTests.vb

```diff
@@ -41,5 +41,13 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Recommendations.St
 <MethodBody>For _
 |</MethodBody>, "Each")
         End Function
+
+        <WorkItem(4946, "http://github.com/dotnet/roslyn/issues/4946")>
+        <Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)>
+        Public Async Function NotInForLoop() As Task
+            Await VerifyRecommendationsAreExactlyAsync(
```

Comment:
Yeah, I was looking for that and couldn't find it. Maybe it's just on the C# side? I'll look again.